package.json

```diff
@@ -29,6 +29,7 @@
     "author": "FlowForge Inc.",
     "license": "Apache-2.0",
     "dependencies": {
+        "@flowforge/nr-theme": "^0.1.3",
```

Comment:
@hardillb - As discussed in Eng mtg (tagged here for quick reference)

We will need to `npm i @flowforge/nr-theme@latest` and push after a manual publish of theme versioned as 1.8.0